Makefile

```diff
@@ -11,6 +11,16 @@ setup:
 testsetup:
 	raco make infra/*.rkt
 
+distribute:
+	mkdir -p fpbench-compiled/
```

Comment:
Kinda ugly name, since it could be confused for some other thing. build? target?